foreign_cc/built_tools/private/built_tools_framework.bzl

```
@@ -44,12 +55,25 @@ def built_tool_rule_impl(ctx, script_lines, out_dir, mnemonic):
     """
 
     root = detect_root(ctx.attr.srcs)
+    lib_name = ctx.attr.name
+    env_prelude = get_env_prelude(ctx, lib_name, [], "")
+
+    cc_toolchain = find_cpp_toolchain(ctx)
+
+    path_prepend_cmd = ""
+    if "win" in os_name(ctx):
+        # Prepend PATH environment variable with the path to the toolchain linker, which prevents MSYS using its linker (/usr/bin/link.exe) rather than the MSVC linker (both are named "link.exe")
+        linker_path = paths.dirname(cc_toolchain.ld_executable)
```

Will it always be the case that this path starts with a drive? I feel like it'd be worth checking for that case but then otherwise doing nothing. Your thoughts?
```suggestion
        linker_path = paths.dirname(cc_toolchain.ld_executable)
        if linker_path[0].is_alpha() and linker_path[1] == ":":
            linker_path = linker_path.replace(linker_path[0:2], "/" + linker_path[0].lower())
```

---

Sure, done :)